build.fsx

```diff
@@ -191,7 +213,10 @@ Target "Default" DoNothing
 "Default"
   ==> "PrepareRelease"
   ==> "Build.Net35"
+  =?> ("Build.NetCore", isDotnetSDKInstalled)
+  =?> ("RunTests.NetCore", isDotnetSDKInstalled)
   ==> "NuGet"
+  =?> ("Nuget.AddNetCore", isDotnetSDKInstalled)

```

Comment:
@eiriktsarpalis it's easy with fake to make the conditional as you wish or change it based on env var, etc.

it's written like that so doesnt break the build when someones doesnt have that installed, that's my main fear.

Np about renaming `Nuget`, good :smile: